components/org.wso2.carbon.identity.api.server.extension.management/org.wso2.carbon.identity.api.server.extension.management.v1/src/main/java/org/wso2/carbon/identity/api/server/extension/management/v1/impl/ExtensionsApiServiceImpl.java

```diff
@@ -88,7 +88,7 @@ public Response getExtensionInfoById(String extensionType, String extensionId) {
 
         validateExtensionType(extensionType);
         try {
-            ExtensionInfo extensionInfo = ExtensionManagementServiceHolder.getInstance().getExtensionManager()
+            ExtensionInfo extensionInfo = ExtensionManagementServiceHolder.getExtensionManager()
```

Comment:
Null check for ExtensionManagementServiceHolder.getExtensionManager() is missing?